File: src/utils/defoldTweens/Tween.ts
/* eslint-disable @typescript-eslint/ban-types */
/* eslint-disable @typescript-eslint/no-extra-non-null-assertion */
/* eslint-disable @typescript-eslint/no-unsafe-member-access */
/* eslint-disable @typescript-eslint/no-non-null-assertion */
/* eslint-disable @typescript-eslint/no-unsafe-return */
import { _newTweensForNode } from "./TweenGUI";

export function tween(obj: string | node | hash) {
    // eslint-disable-next-line @typescript-eslint/no-unsafe-return
    return _newTweensForNode(obj);
}

File: app/utils/defoldTweens/Tween.lua
local ____exports = {}
local ____TweenGUI = require("utils.defoldTweens.TweenGUI")
local _newTweensForNode = ____TweenGUI._newTweensForNode
function ____exports.tween(obj)
    return _newTweensForNode(obj)
end
return ____exports
